### PR TITLE
APS 02 

### PR DESCRIPTION
**Pull request by mahendra-aps**
Please review new changes related to Standing Order, Future Dated, Cross Border, CAF, Transaction filtering in SDK as per Leena Jiras . Please also review a new change where we have separated security and Proto lib from SDK. Security and Proto lib for C# is already published to NuGet. This was done to remove the class conflict between SDK and SDK integration tests in SDK test Csharp. Please let us know if you any queries on this point

### CODE REVIEW

File: core/src/Account.cs [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
+using System.Threading.Tasks;
 using Tokenio.Proto.Common.SecurityProtos;
 using Tokenio.Proto.Common.TransactionProtos;
 using Tokenio.Rpc;
 using static Tokenio.Proto.Common.SecurityProtos.Key.Types;
 using ProtoAccount = Tokenio.Proto.Common.AccountProtos.Account;
 
-namespace Tokenio
-{
+namespace Tokenio {
```
Comment (yuhanniu) [inline core/src/Account.cs]: Why are you changing bracket style?

Comment (priyanshu-ap) [reply core/src/Account.cs]: Done and committed the changes.

File: sdk/src/TokenRequestState.cs [hunk 2]
```diff
@@ -1,24 +1,19 @@
...
-{
+namespace Tokenio {
     [Serializable]
-    public class TokenRequestState
-    {
-        public static TokenRequestState Create(string csrfTokenHash, string state)
-        {
-            return new TokenRequestState
-            {
+    public class TokenRequestState {
```
Comment (yuhanniu) [inline sdk/src/TokenRequestState.cs]: there are still a lot of bracket style changes

File: sdk/src/TokenRequestState.cs [hunk 3]
```diff
@@ -1,24 +1,19 @@
...
+        public static TokenRequestState Create(string csrfTokenHash, string state) {
+            return new TokenRequestState {
                 CsrfTokenHash = csrfTokenHash,
-                InnerState = state
+                    InnerState = state
```
Comment (yuhanniu) [inline sdk/src/TokenRequestState.cs]: Are you using some automatic formatter? Changes are like are unnecessary and make this PR unnecessarily large and messy.

**Pull request closed without merging**